templates/_linux.yaml

```diff
@@ -114,4 +114,9 @@ parameters:
 - name: PVCNAME
   description: Name of the PVC with the disk image
   required: true
-
+- name: SRCPVCNAME
```

Comment:
Can you replace `SRCPVCNAME` with `SRC_PVC_NAME` for better readability (same for namespace)?